jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/MockHttpStream.java

```diff
@@ -33,7 +33,32 @@
 public class MockHttpStream implements HttpStream
 {
     private static final Throwable SUCCEEDED = new Throwable();
-    private static final Content.Chunk DEMAND = Content.Chunk.from(BufferUtil.EMPTY_BUFFER, false);
+    private static final Content.Chunk DEMAND = new Content.Chunk()
```

Comment:
Isn't this `EMPTY`? Why do you need it to be different from `EMPTY`?

Reply:
No, it's a different instance on purpose as it is assigned and compared with `==`.